src/app/pages/auth/forgot-password/forgot-password.page.ts

```diff
@@ -39,6 +40,7 @@ export class ForgotPasswordPage {
 
   private readonly ForgotPasswordHttpService  = inject(ForgotPasswordHttpService );
```

Comment:
cambie el nombre del atributo a miniscula siempre la misma palabra.

```suggestion
  private readonly forgotPasswordHttpService  = inject(ForgotPasswordHttpService );
```